apps/meteor/tests/data/users.helper.js

```diff
@@ -86,3 +83,9 @@ export const setUserActiveStatus = (userId, activeStatus = true) =>
 			})
 			.end(resolve);
 	});
+
+export const setUserStatus = (overrideCredentials = credentials, status = 'online') =>
```

Comment:
```suggestion
export const setUserStatus = (overrideCredentials = credentials, status = UserStatus.ONLINE) =>
```

Reply:
That file doesn't look related to these types of changes

Reply:
I vaguely remember that this was created also as a helper for users related actions like login...so setting a user's status should fall into this area